### ISSUE TITLE
Checkbox readme doc needs elements image

### ISSUE THREAD

**Issue by abumami** [wix/stylable-components#473]
OK, so I know the checkbox is super simple, and doesn't really have elements (that's the plural). But, it does have an element (that's the singular), and therefore should have an elements image (if you want to nitpick, you can call it an "element image") to be consistent with the other component readme docs.

**Comment by arnonkehat** [comment]
Didn't it have one previously?

**Comment by urik9** [comment]
I'm not sure about stylable-components but core3-components checkbox has an element image that we can use:
https://github.com/wixplosives/core3-components/blob/master/specs/checkbox.md

**Comment by abumami** [comment]
I suspect it had an image, but I would have to check archeological ruins to be sure. The core image that @urik9 is referring to is a behavior image and not strictly an elements image.

Seems to me that an older image (if it exists) would be OK, but I can't assume so. I just want to be sure that the image reflects the newest spec.

**Comment by abumami** [comment]
Just did some archeological spelunking (backups, drafts, older versions, etc) and could not find an elements image for checkbox.

**Comment by arnonkehat** [comment]
@urik9 your call. are you happy with what was in c3c? i think it could use standardization with the same app @maksymc is using for these drawings. though i have no clue what that is :)

**Comment by abumami** [comment]
@arnonkehat Good point re: standardizing app for the drawings. I mentioned early on that it drives me crazy that the drawings all have a different look. It seems like everyone is using a different drawing app. Would be nice if we could agree on **one** that satisfies everyone.

**Comment by arnonkehat** [comment]
@maksymc @urik9 @LeonidLevi can we agree on such a tool?
